src/libcore/num/wrapping.rs

```diff
@@ -18,6 +18,7 @@ macro_rules! sh_impl_signed {
                 }
             }
         }
+        forward_ref_binop! { impl Shl, shl for Wrapping<$t>, $f }
```

Comment:
Oh, we probably need a `#[stable(since = ...]` annotation, right?

Reply:
Yeah, you may need to modify the macro definition to allow for that.

Reply:
Ok 👍 What's the Rust version that will release this? 1.38.0?

Reply:
1.39 I believe.

Reply:
@sfackler What feature name should I use?

Reply:
It doesn't matter - `wrapping_ref_ops` or whatever should be fine.